docs/explanations/qrao_explanations.rst

```diff
@@ -0,0 +1,368 @@
+when we explicitly define our quantum relaxation.
+
+A Simple Quantum Relaxation
+---------------------------
+
+Before explicating the full quantum relaxation scheme based on
+single-qubit Quantum Random Access Codes (QRACs), it may be helpful to
+first discuss `a version of quantum
+optimization <https://github.com/Qiskit/qiskit-optimization/blob/main/docs/tutorials/06_examples_max_cut_and_tsp.ipynb>`__
+which users may be more familiar with, but discussed in the language of
+quantum relaxation and rounding.
+
+Consider the embedding
+
+.. math:: F^{(1)}: m \in \\{-1,1\\}^M \mapsto \\{|0\rangle,|1\rangle\\}^{\otimes M},
+
+.. math:: \text{cut}(m) \mapsto \text{Tr}\big(H^{(1)}F^{(1)}(m)\big),\quad  H^{(1)} = \sum_{ij; e_{ij} \in E} \frac{1}{2}(1-Z_i Z_j),
+
+where :math:`Z_i` indicates the single qubit Pauli-Z observable defined
+on the :math:`i`\ ’th qubit and Identity terms on all other qubits. It
+is worth convincing yourself that this transformation is a valid
+relaxation of our problem. In particular:
+
+.. math:: \text{cut}(m) = \text{Tr}\big(H^{(1)}F^{(1)}(m)\big) \quad \forall m \in \\{-1,1\\}^M
+
+This sort of embedding is currently used by many near-term quantum
+optimization algorithms, including many `QAOA and VQE based
+approaches <https://github.com/Qiskit/qiskit-optimization/blob/main/docs/tutorials/03_minimum_eigen_optimizer.ipynb>`__.
+Observe how although the relaxed version of our problem can exactly
+reproduce the objective function cut(:math:`m`) for inputs of the form
+:math:`\\{|0\rangle,|1\rangle\\}^{\otimes M}`, we are also free to
+evaluate :math:`H^{(1)}` using a continuous superposition of such
+states. This stands in analogy to how one might classically relax an
+optimization problem such that they optimize the objective function
+using continuous values.
+
+Crucially, a relaxation is only useful if there is some practical way to
+**round** relaxed solutions back onto the original problem’s set of
+admissible solutions. For this particular quantum relaxation, the
+rounding scheme is simply given by measuring each qubit of our relaxed
+solution in the :math:`Z`-basis. Measurement will project any quantum
+state onto the set of computational basis states, and consequently, onto
+the image of :math:`F^{(1)}`.
+
+Quantum Relaxations via Quantum Random Access Codes (QRACs)
+-----------------------------------------------------------
+
+Quantum Random Access Codes were `first outlined in 1983 by Stephen
+Wiesner
+[2] <http://users.cms.caltech.edu/~vidick/teaching/120_qcrypto/wiesner.pdf>`__
+and were used in the context of communication complexity theory. We will
+not be using QRACs in the way they were originally conceived, instead we
+are co-opting them to define our quantum relaxations. For this reason
+will not provide a full introduction to RACs or QRACs, but encourage
+interested readers to seek out more information about them.
+
+:math:`(1,1,1)`, :math:`(2,1,p)`, and :math:`(3,1,p)` Quantum Random Access Codes
```

Comment:
It would be great if you mention (4, 1, p)-QRAC is proven to be impossible.
https://iopscience.iop.org/article/10.1088/1367-2630/8/8/129